gensim/__main__.py

```diff
@@ -0,0 +1,19 @@
+import platform
+import sys
+import numpy
+import scipy
+import gensim
+import argparse
+
+if __name__ == "__main__":
+
+        parser = argparse.ArgumentParser()
```

Comment:
Where is the description? Should use `__doc__` variable like here https://github.com/RaRe-Technologies/gensim/commit/117d447087b2e9a36fda1279d837e045fefd1558